components/esp_wifi_remote/test/smoke_test/sdkconfig.ci.6

```diff
@@ -0,0 +1 @@
+CONFIG_SLAVE_IDF_TARGET_ESP32C6=y
```

Comment:
Have you considered some alternative to define target under test from CI matrix?

Reply:
I don't think we could actually use github matrix.
the only optiohn is to use ${ENV ...} and set it up from environmnet, but the current method is cleaner IMO

Reply:
Left as is, but renamed configs from numbers to `slave_esp32xx`